recipes/quickstart/finetuning/code-merge-inference.py

```diff
@@ -0,0 +1,117 @@
+import os
+import sys
+import argparse
+from PIL import Image as PIL_Image
+import torch
+from transformers import MllamaForConditionalGeneration, MllamaProcessor
+from accelerate import Accelerator
+from peft import PeftModel  # Make sure to install the `peft` library
+
+accelerator = Accelerator()
+device = accelerator.device
+
+# Constants
+DEFAULT_MODEL = "meta-llama/Llama-3.2-11B-Vision-Instruct"
+
+
+def load_model_and_processor(model_name: str, hf_token: str, finetuning_path: str = None):
```

Comment:
I wonder if we can merge this python script into the existing [multi_modal_infer_gradio_UI](https://github.com/meta-llama/llama-recipes/blob/9f4be9e39adfb9cd2568794e5548b62f139e2bea/recipes/quickstart/inference/local_inference/multi_modal_infer_gradio_UI.py#L28), basically we are adding addition argument to load Lora adaptors from some local path.

Reply:
Yes, we can merge these scripts and make it better than before. If possible, can you merge it as of now?
I'll be out of town this week, so not be able to merge. but yes, we can work on it toghether.

Reply:
Let me know what you think.